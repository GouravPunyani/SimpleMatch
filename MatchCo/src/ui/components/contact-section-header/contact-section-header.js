// @flow

import React from 'react'
import { View, StyleSheet } from 'react-native'
import { Text } from '../..'
import { fontFamily, color, spacing } from '../../theme'

export type Props = {
  text: string
};

export function ContactSectionHeader (props: Props): React.Children {
  return (
    <View style={styles.wrapper}>
      <Text style={styles.text}>{props.text}</Text>
    </View>
  )
}

const styles = StyleSheet.create({
  wrapper: {
    borderBottomWidth: 1,
    borderBottomColor: color.brand,
    paddingBottom: spacing.extraSmall,
    marginBottom: spacing.extraExtraSmall
  },
  text: {
    fontSize: 16,
    fontFamily: fontFamily.sangBlueLightItalic
  }
})
