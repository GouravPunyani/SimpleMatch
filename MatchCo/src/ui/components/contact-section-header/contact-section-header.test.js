// @flow
import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'
import { ContactSectionHeader as Comp } from './contact-section-header'

test('renders correctly', () => {
  const tree = renderer.create(<Comp text='foo' />)
  expect(tree).toMatchSnapshot()
})
