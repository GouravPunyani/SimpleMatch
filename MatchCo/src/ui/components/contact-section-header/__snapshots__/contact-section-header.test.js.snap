// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "borderBottomColor": "#E1CD96",
      "borderBottomWidth": 1,
      "marginBottom": 2,
      "paddingBottom": 4,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "#333333",
          "fontFamily": "Nexa",
          "fontSize": 14,
        },
        Object {
          "fontFamily": "SangBleuBP-LightItalic",
          "fontSize": 16,
        },
      ]
    }
  >
    foo
  </Text>
</View>
`;
